Utility.h

```diff
@@ -1,17 +1,19 @@
 #pragma once
 
+#include <string>
 #include <vector>
 #include <memory>
-#include "Item.h"
-
```

Comment:
Why did you remove this?
- [ ] REVISE

Reply:
Figured only forward definition of struct Item is needed in Utility.h , because the definitions use pointer, reference and std::unique_ptr to Item.